code-buddys.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>code-buddys</title>
+    <link rel="stylesheet" href="styles/index.css" />
+  </head>
+  <body class="app-grid">
+    <header class="headerbig">
+      <h1 class="headerbig__title">code buddys</h1>
+      <span class="header__subtitle">on Monday - 18.07.2020</span>
+    </header>
+    <main class="app-grid__content">
+      <section class="code-buddys-container">
```

Comment:
You could leave out container for prettier BEM. :)

```suggestion
      <section class="code-buddys">
```